spec/features/admin_edits_beer_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+feature "Admin edits information for each individual beer page", %{
+    As an admin, I want to be able to edit
+    the information for all beers cause I’m in charge
+  } do
+
+# Acceptance Criteria
+# I must be signed in
+# I must be able to get here through the details page for a specific beer
+# I must fill in the required fields
+# I must be presented with errors if I fill out the form incorrectly
+
+  scenario 'successfully edits beer' do
+    admin = FactoryGirl.create(:admin)
+    beer = FactoryGirl.create(:beer)
+
+    visit root_path
+    click_link 'Sign In'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.